drivers/dose/include/dose_params.h

```diff
@@ -30,7 +30,7 @@ extern "C" {
  * @{
  */
 #ifndef DOSE_PARAM_UART
-#define DOSE_PARAM_UART         (UART_DEV(0))
+#define DOSE_PARAM_UART         (UART_DEV(1))
 #endif
```

Comment:
Seems like the original contributor was overriding this anyway https://github.com/RIOT-OS/RIOT/pull/10710, do it makes sense to change the default.

Reply:
I was wondering why the console wouldn't work when I ran the test, only to discover that the stdio UART was now occupied by DOSE.

I think this will make testing easier.